test/Lightrail.js

```diff
@@ -7,7 +7,7 @@ var path = require("path")
 require("dotenv").config({path: path.join(__dirname, "..", "build", "test.env")})
 
 describe('Lightrail', () => {
-    it('should instantiate without options', ()=>{
+    it('Instantiates without options', ()=>{
```

Comment:
Reword this one ... Instantiates successfully with default configuration

Reply:
done